dnsapi/dns_dynu.sh

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env sh
+
+#Client ID
+#Dynu_ClientId="0b71cae7-a099-4f6b-8ddf-94571cdb760d"
+#
+#Secret
+#Dynu_Secret="aCUEY4BDCV45KI8CSIC3sp2LKQ9"
+#
```

Comment:
please add your name and link here, so that one can report bugs to you.
Thanks.